doc/start/os-recommendations.rst

```diff
@@ -71,6 +71,8 @@ non-Linux systems but these are not supported by the core Ceph effort.
 - **A**: Ceph provides packages and has done comprehensive tests on the software in them.
 - **B**: Ceph provides packages and has done basic tests on the software in them.
 - **C**: Ceph provides packages only. No tests have been done on these releases.
+- **1**: Testing has been done on Centos 9 starting on version 17.2.8 for
+  Quincy and version 18.2.3 for Reef.
```

Comment:
looks good for quincy, but reef was already an A for centos 9. that part shouldn't need to be qualified

Reply:
Roger that. I'll take it out.

Reply:
> reef was already an A for centos 9. that part shouldn't need to be qualified

Right, my apologies for confusion -- I got too carried away with upgrade suites in my note (we clearly weren't testing _that_ on centos 9).

Since I'm commenting already, I wonder why centos 8 line isn't just being removed?  Keeping around an empty line doesn't make sense to me.